src/test/java/com/faforever/server/geoip/GeoIpServiceTest.java

```diff
@@ -50,4 +50,23 @@ public void lookupCountryLoopbackReturnsEmpty() throws Exception {
     Optional<String> result = instance.lookupCountryCode(InetAddress.getLoopbackAddress());
     assertThat(result.isPresent(), is(false));
   }
+
+  @Test
+  public void lookupTimezone() throws Exception {
+    //Timezone data is not availabe for all ip adresses, alternatively use MaxMind server (108.168.255.243)
```

Comment:
In order to be consistent and thus clean and predictable, add space between `//` and `Timezone`.

Also, please fix the typos the IDE is showing you kindly :P
![image](https://cloud.githubusercontent.com/assets/1864183/24667747/c2341c72-1964-11e7-9e9c-671a9252e7cc.png)